addon/services/events-service.ts

```diff
@@ -43,7 +42,7 @@ export default class EventsService extends Service {
    * A method that returns an Observable that will emit events when a 'matching' message is received.
    *
    * @param {Matcher} matcher - A function that tells if a ResourceEvent should trigger the Observable.
-  */
+   */
   watch(matcher: Matcher<ResourceEvent>): Observable<ResourceEvent> {
     return new Observable(this._onMessageObservers, matcher);
```

Comment:
I know this is not a part of the changes ^^ Just I don't understand a thing here. Why create a new Observable each time ? (it's more for onConnected etc... because here we need to have one by matcher)

Reply:
`Observable` represents one instance of a caller actually looking for a specific event. This design allows to have two callers at different code locations to actually whatch for the same event.

The `ObservableGroup` actually represents the event type `onMessage` `onConnect` `onDisconnect`, and a group dispatch an event to a set of registered observables (modulo the observable matcher). 

Did I answer the question?

Reply:
Yes, I get the concept. It's more on the `onConnected` and `onDisconnected` why return a new Observable each time, where return always the same will do the "same". It's not critical at all, haha.

Reply:
We made the choice that one observable could serve one callback, hence N observables for N Callbacks.